deploy/cephfs/helm/templates/provisioner-clusterrole.yaml

```diff
@@ -10,6 +10,9 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 rules:
+  - apiGroups: [""]
```

Comment:
please also bump up the chart version 
cc @kfox1111

Reply:
```
--- a/deploy/cephfs/helm/Chart.yaml
+++ b/deploy/cephfs/helm/Chart.yaml
@@ -4,7 +4,7 @@ appVersion: "1.0.0"
 description: "Container Storage Interface (CSI) driver,
 provisioner, and attacher for Ceph cephfs"
 name: ceph-csi-cephfs
-version: 0.5.1
+version: 0.5.0
```